spec/models/category_list_spec.rb

```diff
@@ -58,25 +58,31 @@
           .categories
           .find { |x| x.name == public_cat.name }
           .displayable_topics
-          .count,
-      ).to eq(3)
```

Comment:
Testing count here is not reliable because it could be 3 private topics which the user should not see and this spec will still pass.